graph/api/src/main/scala/org/apache/spark/graph/api/PropertyGraph.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.graph.api
+
+import org.apache.spark.sql.{DataFrame, SaveMode}
+
+/**
+ * A Property Graph as defined by the openCypher Property Graph Data Model.
+ *
+ * A graph is always tied to and managed by a [[CypherSession]].
+ * The lifetime of a graph is bound by the session lifetime.
+ *
+ * @see <a href="http://www.opencypher.org/">openCypher project</a>
```

Comment:
Is it a direct link to the property graph model document?

Reply:
I added one initially, but it exceeds the line length. The [Databricks Scala Style Guide](https://github.com/databricks/scala-style-guide#linelength) mentions that more than 100 chars are fine for URLs, but scalastyle still complains. Probably because it's behind an `@see` tag. I re-added it and let's see what CI says.